lib/active_model/serializer/adapter/json.rb

```diff
@@ -9,12 +9,6 @@ def serializable_hash(options = nil)
           options ||= {}
           { root => Attributes.new(serializer).serializable_hash(options) }
         end
-
-        private
-
-        def fragment_cache(cached_hash, non_cached_hash)
-          ActiveModel::Serializer::Adapter::Json::FragmentCache.new.fragment_cache(cached_hash, non_cached_hash)
-        end
```

Comment:
looks like it just moved to here: https://github.com/rails-api/active_model_serializers/pull/1127/files#diff-a03583a7a95fe7db05d78cd7ba222895R74